cmd/minikube/cmd/delete.go

```diff
@@ -239,7 +239,9 @@ func deleteProfile(profile *config.Profile) error {
 		out.T(out.FailureType, "Failed to kill mount process: {{.error}}", out.V{"error": err})
 	}
 
-	deleteHosts(api, cc)
+	if driver.IsVM(cc.Driver) {
```

Comment:
This doesn't cover the `none` case, so perhaps this should be `if !driver.IsKIC`.

It feels weird to special case KIC here, so I'm OK with calling `delete` twice, but so long as the error isn't displayed. I feel like it should be able to interpret the lack of container as "nothing to do" rather than outputting a warning message.

Reply:
sounds good -- I changed it so that if we get an error when getting host status, we assume that the host no longer exists and return